wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -991,4 +984,140 @@ protected function getSpiderID($userAgent) {
 	public function isFirstVisit() {
 		return $this->firstVisit;
 	}
+	
+	/**
+	 * Returns all user sessions for a specific user.
+	 *
+	 * @return      \wcf\system\session\Session[]
+	 * @since       5.4
+	 */
+	public function getUserSessions(User $user): array {
+		if ($user->userID === 0) {
+			throw new \InvalidArgumentException("The given user is a guest.");
+		}
+		
+		$sql = "SELECT          *
+			FROM            wcf".WCF_N."_user_session
+			WHERE           userID = ?";
+		$statement = WCF::getDB()->prepareStatement($sql);
+		$statement->execute([$user->userID]);
+		
+		$sessions = [];
+		
+		while ($row = $statement->fetchArray()) {
```

Comment:
```suggestion
		$sessions = [];
		while ($row = $statement->fetchArray()) {
```

Usual code style does not put an empty line.